wp-content/themes/reconasia/assets/_scss/pages/home.scss

```diff
@@ -2,162 +2,370 @@
+      width: rem(36);
+      height: rem(18);
+      margin-top: rem(4);
+    }
+
+    &-posts {
+      margin-top: rem(32);
     }
 
     .post-block {
+      display: block;
+      padding-bottom: rem(32);
+
+      &:not(:last-child) {
+        border-bottom: 1px solid $color-border-light-115;
+      }
+
       &:first-of-type {
-        @include breakpoint('medium') {
-          grid-template-areas:
-            'img'
-            'title'
-            'excerpt'
-            'authors'
-            'date'
-            'categories';
-          grid-template-columns: 1fr;
-          grid-template-rows: auto;
+        margin-top: 0;
+      }
+
+      &__img {
+        display: none;
+      }
+
+      &__title,
+      .post-meta__date {
+        margin-bottom: rem(16);
+      }
+
+      &__excerpt {
+        margin-bottom: rem(8);
+      }
+
+      /* stylelint-disable-next-line */
+      .post-meta__categories li:not(:last-of-type)::after {
+        border-right: 1px solid $color-border-light-130;
+      }
+
+      @include breakpoint('large') {
+        &:last-child {
+          border-bottom: 0;
         }
       }
     }
   }
 
-  &__series {
-    --padding-top: #{rem(20)};
-    --padding-sides: 0;
+  &__featured-primary {
+    @include csis-block-full-width-wrapper();
+    position: relative;
+    grid-area: header;
+    padding: rem(80) 6vw rem(16);
+
+    @include breakpoint('small') {
+      margin-bottom: rem(40);
+      padding-top: rem(160);
+      padding-bottom: rem(32);
+    }
 
-    /* prettier-ignore */
-    $gradient: linear-gradient(180deg, #f1f4f4 0%, rgba(245, 246, 247, 0.88) 100%);
+    @include breakpoint('medium') {
+      padding-bottom: rem(40);
+    }
 
-    padding: var(--padding-top) var(--padding-sides) 0;
-    color: $color-black-100;
-    background: $gradient;
+    @include breakpoint('large') {
+      padding-bottom: rem(56);
+    }
 
-    @include breakpoint($break: 'medium', $until: 'large') {
-      display: grid;
-      grid-column-gap: rem(40);
-      grid-template-areas:
-        'title title'
-        'desc desc'
-        'articles img'
-        'archive archive';
-      grid-template-columns: 2fr 1fr;
+    @include breakpoint('xxlarge') {
+      padding-right: rem(80);
+      padding-left: rem(80);
     }
 
-    @include breakpoint($break: 'large', $dir: 'max-width') {
-      @include full-width-background($gradient);
+    &-label {
+      @extend %text-style-label-small;
+      display: flex;
+      align-items: flex-end;
+      width: fit-content;
+      height: rem(30);
+      padding-right: rem(22);
+      padding-left: rem(16);
+      font-weight: normal;
+      text-transform: uppercase;
+      background: $color-bg-dark-290;
+      clip-path: polygon(0 0, 85% 0, 100% 100%, 0% 100%);
+
+      @include breakpoint('medium') {
+        padding-right: rem(32);
+        padding-left: rem(24);
+      }
+
+      @include breakpoint('xxlarge') {
+        margin-left: calc((100vw - 1400px) / 2);
+      }
```

Comment:
Almost there! To take into account the scrollbar:

margin-left: calc(((100vw - 1400px)  / 2) - var(--scrollbarHalf));